src/server/game/Handlers/MailHandler.cpp

```diff
@@ -22,8 +22,11 @@ bool WorldSession::CanOpenMailBox(uint64 guid)
 {
     if (guid == _player->GetGUID())
     {
-        sLog->outError("%s attempt open mailbox in cheating way.", _player->GetName().c_str());
-        return false;
+        if (!SEC_MODERATOR)
```

Comment:
if (!SEC_MODERATOR) ??? wut

This should be:
```
if (_player->GetSession()->GetSecurity() < SEC_MODERATOR)
```

Reply:
Does it matter? O.o
Works with !SEC_MODERATOR

Reply:
![image](https://user-images.githubusercontent.com/24550914/79250012-61340680-7e7e-11ea-98e9-427e5909a132.png)
With Account Security 1,2,3,4 (MOD,GM.ADMIN,CONSOLE)
With Account Security 0 regular mailbox works as intended and nothing is broken

Reply:
It matters. Right now your if statement looks like this:
```
if (!1)
```
It will always return true; 
meaning anyone regardless of their gmlevel can open the mailbox.

Reply:
I think SEC_MODERATOR includes the above too, because without this change, .mailbox don't work for anything, and now it works for all

Reply:
@lineagedr is right

Reply:
Pls explain why so I can learn. Because every test I make this works